cdap-app-fabric-tests/src/test/java/io/cdap/cdap/internal/app/services/http/handlers/WorkflowStatsSLAHttpHandlerTest.java

```diff
@@ -63,7 +64,7 @@
  */
 public class WorkflowStatsSLAHttpHandlerTest extends AppFabricTestBase {
 
-  private static final ApplicationId WORKFLOW_APP = NamespaceId.DEFAULT.app("WorkflowApp");
+  private static ApplicationId workflowApp = NamespaceId.DEFAULT.app("WorkflowApp");
```

Comment:
+1. all static members should be final

Reply:
because at L220 - we reset workflow-id to point to latest version and not -SNAPSHOT (these tests call the services directly not the endpoints) so we need to point to the right version

Reply:
Anything you want to change should not be static. Also generally try use variables and not fields for mutable values

Reply:
got it, not making this field static